### DISCUSSION TITLE
Brainstorm 3D visualizers and tools for making them

### DISCUSSION THREAD

**Discussion by liammulh** [numberscope/frontscope#312] [Ideas]
We might eventually want some 3D visualizers.

Library Ideas:
- p5 has some built-in 3D capabilities
- three.js is a very popular 3D library for JavaScript
- other 3D libs?
- "3D SVGs": https://doc.x3dom.org/tutorials/animationInteraction/transformations/example.html

**Comment by gwhitney** [comment]
They should be possible now using P5GLVisualizer, but leaving this discussion open for other ideas that might be worth implementing.
